tests/test_regex_identifier.py

```diff
@@ -10,303 +10,303 @@ def test_regex_successfully_parses():
 
 def test_regex_runs():
     r = regex_identifier.RegexIdentifier()
-    res = r.check(["DANHz6EQVoWyZ9rER56DwTXHWUxfkv9k2o"])
-    assert "Dogecoin (DOGE) Wallet Address" in res[0]["Regex Pattern"]["Name"]
+    res = r.check({"test": ["DANHz6EQVoWyZ9rER56DwTXHWUxfkv9k2o"]})
```

Comment:
Why does every API input need to be in a dictionary? What are the values of this dictionary? What is `test`?

Reply:
Sooo I firstly assigned every text to a file `{"filename": ["text"]}` and this part was then submitted to check(), so this is why I changed all the tests as it required a dictionary with a filename. I have already changed this and will push a commit with some of the other changes - now it assigns matches to a filename AFTER check() comes back with the results, so tests were reverted back. Did not notice this yesterday :')

Reply:
Yes! I have changed this, so no changes are needed for the API and also tests were reverted back as they no longer need to be dictionaries :D